### PR TITLE
[040-node-manager] Node manager fix FAQ

### PR DESCRIPTION
**Pull request by cds111**
## Description
Replacing deprecated `spec.cloudInstances.standbyHolder.notHeldResources` with `spec.cloudInstances.standbyHolder.overprovisioningRate` in the FAQ.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: node-manager
type: chore
summary: |
  Replace `spec.cloudInstances.standbyHolder.notHeldResources` with `spec.cloudInstances.standbyHolder.overprovisioningRate`.
impact_level: low
```

### CODE REVIEW

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -413,7 +413,7 @@ nodeTemplate:
 Необходимые настройки целевой `NodeGroup` будут следующие:
 
 1. Указать абсолютное количество *подогретых* узлов (или процент от максимального количества узлов в этой группе) в параметре `cloudInstances.standby`.
-1. При наличии дополнительных служебных компонентов (не обслуживаемых Deckhouse, например DaemonSet `filebeat`) для этих узлов — задать их суммарное потребление ресурсов в параметре `standbyHolder.notHeldResources`.
+1. При наличии дополнительных служебных компонентов (не обслуживаемых Deckhouse, например DaemonSet `filebeat`) для этих узлов — задать их процентное потребление ресурсов от общего количества ресурсов ноды в параметре `standbyHolder.overprovisioningRate`.
```
Comment (Zhbert) [inline modules/040-node-manager/docs/FAQ_RU.md:416]:
```suggestion
1. При наличии дополнительных служебных компонентов (не обслуживаемых Deckhouse, например DaemonSet `filebeat`) для этих узлов — задать их процентное потребление ресурсов от общего количества ресурсов узла в параметре `standbyHolder.overprovisioningRate`.
```

**Review by Zhbert — APPROVED** [review]

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -413,7 +413,7 @@ nodeTemplate:
 Необходимые настройки целевой `NodeGroup` будут следующие:
 
 1. Указать абсолютное количество *подогретых* узлов (или процент от максимального количества узлов в этой группе) в параметре `cloudInstances.standby`.
-1. При наличии дополнительных служебных компонентов (не обслуживаемых Deckhouse, например DaemonSet `filebeat`) для этих узлов — задать их суммарное потребление ресурсов в параметре `standbyHolder.notHeldResources`.
+1. При наличии дополнительных служебных компонентов (не обслуживаемых Deckhouse, например DaemonSet `filebeat`) для этих узлов — задать их процентное потребление ресурсов от общего количества ресурсов узла в параметре `standbyHolder.overprovisioningRate`.
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ_RU.md:416]:
```suggestion
1. При наличии на узлах дополнительных служебных компонентов, не обслуживаемых Deckhouse (например, DaemonSet `filebeat`), задать их процентное потребление ресурсов узла можно в параметре `standbyHolder.overprovisioningRate`.
```

File: modules/040-node-manager/docs/FAQ.md [hunk 3]
```diff
@@ -407,7 +407,7 @@ The most efficient way is to have some extra nodes "ready". In this case, you ca
 Here is how you should configure the target `NodeGroup`:
 
 1. Specify the number of "ready" nodes (or a percentage of the maximum number of nodes in the group) using the `cloudInstances.standby` paramter.
-1. If there are additional service components (not maintained by Deckhouse, such as `filebeat` DaemonSet) for these nodes, you need to specify their combined resource consumption via the `standbyHolder.notHeldResources` parameter.
+1. If there are additional service components (not maintained by Deckhouse, such as `filebeat` DaemonSet) for these nodes, you need to specify percentage of their combined resource consumption relative to the total node resources in the parameter `standbyHolder.overprovisioningRate`.
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ.md:410]:
```suggestion
1. При наличии на узлах дополнительных служебных компонентов, не обслуживаемых Deckhouse (например, DaemonSet `filebeat`), задать их процентное потребление ресурсов узла можно в параметре `standbyHolder.overprovisioningRate`.
```
@kirkonru Translate please.

Comment (kirkonru) [reply modules/040-node-manager/docs/FAQ.md:410]: @z9r5 done 😎

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9435](https://github.com/deckhouse/deckhouse/pull/9435) to the branch [release-1.63](https://github.com/deckhouse/deckhouse/tree/release-1.63) successful!

**Pull request merged**